### DISCUSSION TITLE
Twitch stream copy

### DISCUSSION THREAD

**Discussion by kleo** [linux-3ds/arm9linuxfw#14] [General]
I checked @nickdesaulniers Twitch channel and the **Building Linux for the Nintendo 3DS Linux Kernel Development** video is no longer available. 

I have a copy and I'm asking permission if it is okay to upload it to my personal Youtube channel?

**Comment by nickdesaulniers** [comment]
cool! Yeah, go for it!

**Reply by kleo** [reply]
[Building Linux for the Nintendo 3DS](https://youtu.be/mTo8yb6q4Lw)
